code-reviews/recipes/Java.md

```diff
@@ -0,0 +1,30 @@
+# Java Code Reviews
+
+## Java Style Guide
+
+[CSE](../../CSE.md) developers generally follow the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html).
+
+## Code Analysis
```

Comment:
Should we change to Linting to follow the other recipes?

Reply:
I think we are only referencing analyzers, so some of the recipes would be better off with Code Analysis, but I think the definition of Linter vs. Analyzers are very close anyway and used interchangeably when I look through the web. Maybe we can use "Linting / Analyzing" as a compromise and update the other sections? @TessFerrandez

Reply:
I'm fine with that approach as I do think there is a lot of overlap.  If @TessFerrandez agrees, I can update that here and then create an issue and update the other docs as well.

Reply:
@TessFerrandez, I would like to close this out today. If you don't agree with the suggestion pls chime in.

Reply:
sounds good to me - we had this discussion on the C# recipe and IIRC you (or someone else) wanted to change to Linters...  as long as it's consistent I am good with either